refresh_compile_commands.bzl

```diff
@@ -100,6 +102,7 @@ _expand_template = rule(
         "labels_to_flags": attr.string_dict(mandatory = True),  # string keys instead of label_keyed because Bazel doesn't support parsing wildcard target patterns (..., *, :all) in BUILD attributes.
         "exclude_external_sources": attr.bool(default = False),
         "exclude_headers": attr.string(values = ["all", "external", ""]), # "" needed only for compatibility with Bazel < 3.6.0
+        "enable_swift": attr.bool(default = False),
```

Comment:
What do you think about just always having swift enabled, eliminating the need for configuration and branching?

I'm asking because swift extraction should be really fast, right, since there's no need for header extraction. Normal clangd doesn't crash or otherwise error out if there are swift commands, right?

You're the expert here, but it seems from the outside like this is worth having on all the time :)

Reply:
This flag designed for some other tools like [infer](https://fbinfer.com)
In bilibili we are using it for static analysis and it didn't support swift yet...

Reply:
Ah! Got it. And to confirm, the static analysis tools crash, rather than gracefully ignoring the swift files? Bummer.

Reply:
How about it we at least turn swift on by default?
Perhaps "exclude_swift" defaulting to False?
(We should make sure to eventually document this in the README and file docstring)

Reply:
It might also be worth submitting bugs to the tools that crash on swift files? They're going to see more and more of those inputs as the years roll on, I think.

Reply:
https://github.com/xinzhengzhang/bazel-compile-commands-extractor/commit/22317ce83a6dd76673efc11ac91b39630d8cb135

Reply:
Sounds good. But if it removes the need for the file filter, I'd still be happy with keeping an `exclude_swift` flag (off by default) for you, until those tools become tolerant to swift files?
(They do crash, yes, or otherwise issue errors that interfere with their use?)

Reply:
Done 
https://github.com/xinzhengzhang/bazel-compile-commands-extractor/commit/047daf96b94fd1cb46453336a9dee5f9d74a023a